### DISCUSSION TITLE
Setting plotLog2FC parameter to TRUE for plotMarkerHeatmap.

### DISCUSSION THREAD

**Discussion by sylestiel** [GreenleafLab/ArchR#1526] [General]
When using the plotMarkerHeatmap function plotLog2FC = FALSE parameter is set to default. If I were to change that to TRUE when querying seMarker for Clusters is the output still valid?

**Comment by rcorces** [comment]
The parameter definition for `plotLog2FC` was missing so I added it via https://github.com/GreenleafLab/ArchR/commit/627a6b12a3cd057a460fb7ae50939909ee331dc3

The output is valid, regardless of the parameters. But the output is different and you should understand why its different.

**Reply by sylestiel** [reply]
Thank you for clarifying that.
